docs/element-templates/README.md

```diff
@@ -368,8 +393,9 @@ As of Camunda Modeler `v1.11.0` we support special scoped bindings that allow yo
     "bpmn:Task"
   ],
   "properties": [],
-  "scopes": {
-    "camunda:Connector": {
+  "scopes": [
```

Comment:
I decided to simply showcase the new style via arrays. If you think we should add the old way and say we still support it in some cases, I'd fine.

Reply:
:+1: